translations/messages.fr.xlf

```diff
@@ -1070,26 +1074,65 @@
+        <target>Les maisons à l'extrême gauche et droite, non alignées sur les autres, sont les Kalah, le vôtre est à gauche, celui de l'adversaire à droite. Quand vous distrubez, avant de passer de votre dernière maison à la première de l'adversaire, vous devez déposer une graine dans votre Kalah. Mais vous ne devrai pas distribuer dans le Kalah adverse. Quand vous capturez, les graines sont déposées dans votre Kalah.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé. Faite un mouveent qui apsse par votre Kalah puis par les maisons adverses.</target>
+      </trans-unit>
+      <trans-unit id="108833396164233870" datatype="html">
+        <source>As you see, three houses have been fed in addition to your Kalah.</source>
+        <target>Voilà, trois maisons on été remplies en plus de votre Kalah.</target>
+      </trans-unit>
+      <trans-unit id="7327320572157893918" datatype="html">
+        <source>Failed. Choose the three leftmost house on the bottom.</source>
+        <target>Raté. Distribuez une des trois maisons du bas les plus à gauche.</target>
+      </trans-unit>
+      <trans-unit id="6340571023242332704" datatype="html">
+        <source>The Kalah (2/2)</source>
+        <target>Le Kalah (2/2)</target>
+      </trans-unit>
+      <trans-unit id="8979647594925459000" datatype="html">
+        <source>When ending in the Kalah, you must distribute again.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, play the house that end up in the Kalah then do a second move!</source>
+        <target>Quand votre distribution fini dans le Kalah, vous êtes authorisé à distribuer à nouveau.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faite la distribution qui termine dans votre Kalah, puis une seconde distribution!</target>
+      </trans-unit>
+      <trans-unit id="1468463450299215954" datatype="html">
+        <source>This move only distributed one house, do one distribution that ends in the Kalah, then do a second one!</source>
+        <target>Ce coup ne distribue qu'une seule maison, faites une distribution qui finit dans le Kalah puis une deuxième!</target>
+      </trans-unit>
+      <trans-unit id="723370864893318309" datatype="html">
+        <source>When the last seed of a distribution ends up in one of your empty houses, if the opposite house is filled, then you capture both houses. On this board, such a move is possible.&lt;br/&gt;&lt;br/&gt;You&apos;re playing Dark, do a capture!</source>
+        <target>Quand votre dernière graine fini dans une de vos maisons vides, si la maison au dessus est remplie, alors vous capturez le graines de ces deux maisons. Sur ce plateau, c'est faisable.&lt;br/&gt;&lt;br/&gt;Vous jouez Foncé, faites une capture!</target>
```

Comment:
espace avant !
et pour ce step, le "fail" n'est pas traduit : You did not capture, try again!